templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/modeling_tf_{{cookiecutter.lowercase_modelname}}.py

```diff
@@ -460,6 +460,20 @@ def build(self, input_shape):
         self.bias = self.add_weight(shape=(self.vocab_size,), initializer="zeros", trainable=True, name="bias")
```

Comment:
Could you update the Seq2Seq Templates as well? => this would be veeery helpful for the TF Bart split PR

Reply:
And I'm afraid the `test_modeling_tf_{{cok...` as well